Open.Nat/AssemblyInfoAdditions.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
Is this new file necessary? I see the same is specified in the Open.Nat.csproj file.

Reply:
This is true for the most part, but the caveat was your test cases use internal functions and/or classes, I had to add the file in order to add InternalsVisibleTo but that is the only statement in there. The rest of the metadata shifted to the csproj file.